InOut/CMakeLists.txt

```diff
@@ -70,14 +63,16 @@ endif()
 # endif()
 
 if(APPLE)
-#    check_deps(USE_COREMIDI APPLE COREMIDI_INCLUDE_PATH COREMIDI_LIBRARY COREFOUNDATION_LIBRARY)
-#   check_deps(USE_AUDIOUNIT APPLE COREAUDIO_INCLUDE_PATH COREAUDIO_LIBRARY AUDIOUNIT_INCLUDE_PATH AUDIOUNIT_LIBRARY COREFOUNDATION_LIBRARY)
+    check_deps(USE_COREMIDI APPLE CoreMIDI_FOUND CoreMIDI_FOUND CoreFoundation_FOUND)
+    check_deps(USE_AUDIOUNIT APPLE CoreAudio_FOUND AudioUnit_FOUND CoreFoundation_FOUND)
+    if(USE_COREMIDI)
         make_plugin(cmidi cmidi.c)
-        target_include_directories(cmidi PRIVATE ${COREMIDI_INCLUDE_PATH})
-        target_link_libraries(cmidi PRIVATE ${COREMIDI_LIBRARY} ${COREFOUNDATION_LIBRARY})
+        target_link_libraries(cmidi PRIVATE CoreMIDI::coremidi CoreFoundation::corefoundation)
+    endif()
+    if (USE_AUDIOUNIT)
         make_plugin(rtauhal rtauhal.c)
-        target_include_directories(rtauhal PRIVATE ${AUDIOUNIT_INCLUDE_PATH})
-        target_link_libraries(rtauhal PRIVATE ${AUDIOUNIT_LIBRARY} ${COREFOUNDATION_LIBRARY} ${COREAUDIO_LIBRARY})
+        target_link_libraries(rtauhal PUBLIC AudioUnit::audiounit CoreFoundation::corefoundation CoreAudio::coreaudio)
```

Comment:
Is this really necessary to set as public? Afaik we don't expose the header for audiounit in libcsound so shouldn't need to do that? I could be wrong

Reply:
Oops, changed to PRIVATE, thanks!